spec/support/database_cleaner.rb

```diff
@@ -0,0 +1,14 @@
+require 'database_cleaner'
+
+RSpec.configure do |config|
+  config.before(:suite) do
+    DatabaseCleaner.strategy = :transaction
+    DatabaseCleaner.clean_with(:truncation)
+  end
+
+  config.around(:each) do |example|
```

Comment:
I don't think we should add this hook for every example in the test suite: could significantly slow things down with these hooks. I would suggest adding this only to spec files that require this.

Reply:
This file is  actually `required` only in the spec that needs this. And since we're running our tests by separate invocations of `rspec` then this actually works only for that spec.

However implementing this using [filters](https://relishapp.com/rspec/rspec-core/docs/hooks/filters) would make it more futureproof WDYT ?

Reply:
I do really like the look of filters, particularly because it allows us to selectively apply hooks like this. That would be an improvement.

Reply:
Cool I'll change this to filters then.